tools/ci_build/github/windows/eager/requirements.txt

```diff
@@ -0,0 +1,9 @@
+setuptools
+wheel
+numpy
+ninja
+cmake
```

Comment:
Is ninja needed?
Please remove cmake because the machine has already installed it at a different location and put it in PATH. It's better to avoid having two copies in PATH.

Reply:
Removed